src/Symfony/Component/Translation/Bridge/Crowdin/CrowdinProvider.php

```diff
@@ -128,39 +124,37 @@ public function read(array $domains, array $locales): TranslatorBag
                     $response = $this->downloadSourceFile($fileId);
                 }
 
-                $responses[] = [
-                    'response' => $response,
-                    'locale' => $locale,
-                    'domain' => $domain,
-                ];
+                $responses[] = [$response, $locale, $domain];
             }
         }
 
-        foreach ($responses as $responseData) {
-            /** @var ResponseInterface $response */
-            $response = $responseData['response'];
-
+        /* @var ResponseInterface $response */
+        $downloads = [];
+        foreach ($responses as [$response, $locale, $domain]) {
             if (204 === $response->getStatusCode()) {
-                $this->logger->error(sprintf('No content in exported file. Message: "%s".', $response->getContent(false)));
+                $this->logger->error(sprintf('No content in exported file: "%s".', $response->getContent(false)));
 
                 continue;
             }
 
             if (200 !== $response->getStatusCode()) {
-                $this->logger->error(sprintf('Unable to export file. Message: "%s".', $response->getContent(false)));
+                $this->logger->error(sprintf('Unable to export file: "%s".', $response->getContent(false)));
 
                 continue;
             }
 
-            $exportResponse = $this->filesDownloader->request('GET', $response->toArray()['data']['url']);
+            $response = $this->client->request('GET', $response->toArray()['data']['url']);
+            $downloads[] = [$response, $locale, $domain];
+        }
 
-            if (200 !== $exportResponse->getStatusCode()) {
-                $this->logger->error(sprintf('Unable to download file content. Message: "%s".', $response->getContent(false)));
```

Comment:
I suppose using `$response` instead of `$exportResponse` here was a bug, isn't it?